tests/dhcp_server/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from tests.common.utilities import wait_until
+
+DHCP_SERVER_CONTAINER_NAME = "dhcp_server"
+DHCP_SERVER_FEATRUE_NAME = "dhcp_server"
+
+
+@pytest.fixture(scope="module", autouse=True)
+def dhcp_server_setup_teardown(duthost):
+    features_state, _ = duthost.get_feature_status()
+    if DHCP_SERVER_FEATRUE_NAME not in DHCP_SERVER_FEATRUE_NAME:
```

Comment:
if DHCP_SERVER_FEATRUE_NAME not in features_state?

Reply:
Good catch, fixed.